src/deployment.jl

```diff
@@ -1,41 +1,59 @@
 """
-    deployment_area(n_corals::Int64, max_n_corals::Int64, density::Float64, target_areas::Vector{Float64})::Tuple{Float64,Float64}
+    rme_version_info()::@NamedTuple{major::Int64, minor::Int64, patch::Int64}
+
+Get RME version
+"""
+function rme_version_info()::@NamedTuple{major::Int64, minor::Int64, patch::Int64}
+    rme_ver = @RME version()::Cstring
+    rme_ver = parse.(Int64, split(rme_ver, '.'))
```

Comment:
No change required here, just nothing this is type unstable (going from string to vector of ints).